### ISSUE TITLE
 Edit the settings of shell, i.e: #39 (Fix)

### ISSUE THREAD

**Issue by jackie1santana** [flathub/com.visualstudio.code#244]
Edit the settings of shell, i.e:

```json
"terminal.integrated.shell.linux": "flatpak-spawn",
"terminal.integrated.shellArgs.linux": ["--host", "bash"],
```

_Originally posted by @ricvelozo in https://github.com/flathub/com.visualstudio.code/issues/39#issuecomment-505747660_

**Comment by jackie1santana** [comment]
**Above is now depreciated.** _This should do the trick:_
```

  "terminal.integrated.defaultProfile.linux": "zsh",
    "terminal.integrated.profiles.linux": {
        "zsh": {
          "path": "/usr/bin/env",
          "args": ["--", "flatpak-spawn", "--host", "--env=TERM=xterm-256color", "zsh"]
        }

    }
```

**Comment by jackie1santana** [comment]
this fixes #39
